### ISSUE TITLE
FR: Allow block referencing via drag-n-drop from query results

### ISSUE THREAD

**Issue by pcuellar** [nothingislost/obsidian-query-control#24]
Terrific work on this plugin first and foremost. I'd like to suggest the possibility to do the equivalent of `![[file#^my_text]]` by dragging and dropping things from search results

**Comment by huachuman** [comment]
Yes this would be an absolutely amazing feature. I tried this not knowing what to expect and was a little disappointed it didn't work.
